wp-property.php

```diff
@@ -34,6 +34,14 @@
   define( 'WPP_URL', plugin_dir_url( __FILE__ ) . 'static/' );
 }
 
+if( !defined( 'WPP_VENDOR_URL' ) ) {
+  define( 'WPP_VENDOR_URL', plugin_dir_url( __FILE__ ) . 'vendor/' );
+}
+
+if( !defined( 'WPP_VENDOR_Path' ) ) {
+  define( 'WPP_VENDOR_Path', plugin_dir_path( __FILE__ ) . 'vendor/' );
+}
+
```

Comment:
@mehulkaklotar please, use the following method of Singleton object to get URL or DIR path instead of adding new constants: 

* `ud_get_wp_property()->path( 'vendor/path_to_directory/file.js', 'dir' ); // to get absolute path to file`
* `ud_get_wp_property()->path( 'vendor/path_to_directory/file.js', 'url' ); // to get absolute URL of file`

Thanks!